packages/salesforcedx-vscode-apex/package.json

```diff
@@ -75,11 +75,13 @@
       {
         "language": "apex",
         "scopeName": "source.apex",
-        "path": "./syntaxes/apex.tmLanguage"
+        "path":
+          "./node_modules/@salesforce/apex-tmlanguage/grammars/apex.tmLanguage"
```

Comment:
@lcampos - Could you do sanity check on the salesforcedx-vscode-core .vsix from CircleCI once they are ready? I just want to make sure that when we run `vscode:package` that the node_modules for this is packaged properly and not pruned away.

Reply:
Everything looks good (verified/tested with the circleci artifact), I'll merge this now.